ci/kubetest/test_kafka_service_nodeport.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from utils import cleanup_k8s, get_kafka_service_spec, helm_install, wait_for_pods_to_be_ready
+
+HELM_INSTALL_CMD = """
+helm upgrade \
+    --install \
+    -f ../../ci/values/kubetest/test_kafka_service_nodeport.yaml \
+    --timeout 30m \
+    --create-namespace \
+    --namespace posthog \
+    posthog ../../charts/posthog \
+    --wait-for-jobs \
+    --wait
+"""
+
+
+@pytest.fixture
+def setup(kube):
+    cleanup_k8s()
+    helm_install(HELM_INSTALL_CMD)
+    wait_for_pods_to_be_ready(kube)
+
+
+def test_helm_install(setup, kube):
+    pass
+
+
+def test_kafka_service(kube):
+    service = get_kafka_service_spec(kube)
+    assert service.type == "NodePort", "Kafka service type is {}".format(service.type)
```

Comment:
Does this give us enough information that we can actually connect to the service from the relevant pods?

I assume not right?

I think it would be worthwhile to smoke test this by:
- Spin up a web pod
- Test accessibility either via https://github.com/PostHog/posthog/pull/8258 or via some other method that actually connects to the kafka cluster.

Reply:
100% the PR is still a WIP so I was going to iterate further on this 😄 

As of now, this test doesn't make sense as kubetest as we are testing something that we should get as "granted" from the upstream chart. I also don't think we should have a unit test specifically for this for the same reason.

If we consider adding as well some additional test condition (like the one you proposed) I think it might make sense.

Reply:
Removed